pynamodb/models.py

```diff
@@ -554,10 +464,9 @@ def count(cls,
         :param consistent_read: If True, a consistent read is performed
         :param index_name: If set, then this index is used
         :param rate_limit: If set then consumed capacity will be limited to this amount per second
-        :param filters: A dictionary of filters to be used in the query. Requires a hash_key to be passed.
         """
         if hash_key is None:
-            if filters:
+            if filter_condition is not None:
                 raise ValueError('A hash_key must be given to use filters')
             return cls.describe_table().get(ITEM_COUNT)
```

Comment:
probably not an issue right now, but this seems broken -- IMHO the right thing to do here is to provide either a filtered query or a filtered scan -- not to use the (what i think is several hours delayed?) table description

Reply:
Yep, this is definitely a problem and a common confusion when using `.count`. Happy to consider changing this API for the 4.0.0 release or we wait until a future one.